pkg/iohelpers/count_reader.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * umoci: Umoci Modifies Open Containers' Images
+ * Copyright (C) 2016-2025 SUSE LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package iohelpers
+
+import (
+	"io"
+)
+
+// An io.Reader wrapper that counts how many bytes were read from the
```

Comment:
`read from the` what? :)

Reply:
Oops, fixed. It seems that our `golint` is not running properly either, since there were some other `golint` issues I caught when running locally...